splice_access_api/src/main/java/com/splicemachine/access/api/FileInfo.java

```diff
@@ -26,21 +26,33 @@
 
     boolean isDirectory();
 
+    /// returns true if isDirectory() and (directory is empty or only contains one file _SUCCESS)
```

Comment:
This should probably be a javadoc comment, you need /** */ for that

Reply:
ok